k8s/jobs/django-migrate-job.yml

```
@@ -11,7 +11,7 @@ spec:
       containers:
         - name: django-migrate
           image: registry.digitalocean.com/mindjunkies/django-mindjunkies-web:latest
-          command: [ "/bin/bash", "-c", "/app/collectstatic.sh" ]
+          command: [ "/bin/bash", "-c", "/app/migrate.sh" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)